docs/README.md

```diff
@@ -4,44 +4,56 @@ Autometa documentation
 Building the documentation
 --------------------------
 
-https://github.com/matplotlib/matplotlib/blob/master/doc/devel/documenting_mpl.rst
-
-This links to Matplotlib how to buikd documentation page.
-
 This file will detail how to download the docs and contribute to them.
 What files to install and how to run
 
 Installation
 ------------
-Install it in the conda environment
+
+Each script in the source will be run with the help flag to automatically place,
+usage information for the respective script. Therefore, the packages necessary to
+run the script need to be available. In order to perform the build, the `autometa`
+conda environment needs to be active and additional packages need to be installed.
 
 ```bash
-conda install -c conda-forge sphinxcontrib-programoutput sphinx sphinx_rtd_theme
+conda install -n autometa -c conda-forge sphinxcontrib-programoutput sphinx sphinx_rtd_theme
```

Comment:
Should we specify `conda activate autometa` ? For newbies like me specifying it explicitly helps a lot

Reply:
Yeah, that's a good idea